.github/workflow_templates/build-and-test_release.yml

```diff
@@ -121,7 +121,7 @@ jobs:
     needs:
       - git_info
       - build_fe
-{!{ tmpl.Exec "tests_template" (slice $ctx "matrix" "build_fe") | strings.Indent 4 }!}
+{!{ tmpl.Exec "tests_before_build_template" (slice $ctx "matrix") | strings.Indent 4 }!}
```

Comment:
It seems build_fe is redundant now. (Github stops me to suggest here)

Reply:
tests are run before the ce/ee/fe build, so a third parameter is not needed.